components/button/button-shared-styles.js

```diff
@@ -0,0 +1,29 @@
+import { css } from 'lit-element/lit-element.js';
+
+export const buttonSharedStyles = css`
```

Comment:
Are we wanting these to be only the styles that are the same for all the button types?  Or should there be a `background-color` set here (for example), that is overwritten by some but not all of the button types?

Reply:
I _think_ just the styles that are common for all button types.  If we include other things, we'll need to decide which one gets to be "default".  I could be convinced otherwise though.  ;)

Reply:
Yeah, I'm not confident enough to want to do any convincing 😛.  It depends on if we want this to contain truly shared styles (which could lead to lots of duplicate declarations in the other style files), or the "main" style (which could lead to lots of overrides if the other button types need to "undo" a style).  This will be really important for `inputs` as well, so we should try to be consistent... 🤔